### PR TITLE
Import mdn/browser-compat-data. Fix #102

### LINKED ISSUE #102: Adopt browser-compat-data to show compatibility in hover / completion
https://github.com/mdn/browser-compat-data

### PR DESCRIPTION
**Pull request by octref**
- Add `mdn/browser-compat-data` as a dev dependency
- Use bcd data whenever it's available

Cosmetics:
- Made all js scripts under `/build` the same style
- Switch to const/let from var

### CODE REVIEW

File: build/generate_browserjs.js [hunk 1]
```diff
@@ -199,196 +198,198 @@ var otherColors = {
...
 }
 
-var parser = new xml2js.Parser({explicitArray : false});
-var schemaFileName= 'css-schema.xml';
+const parser = new xml2js.Parser({ explicitArray: false })
+const schemaFileName = 'css-schema.xml'
 
-var { buildPropertiesWithMDNData } = require('./mdn-data-importer')
+const { buildPropertiesWithMDNData } = require('./mdn-data-importer')
+const { addBCPDataToProperties } = require('./mdn-browser-compat-data-importer')
```
Comment (connorshea) [inline build/generate_browserjs.js]: `BCP`?

File: build/mdn-browser-compat-data-importer.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+ *       {
+ *         "prefix": "-moz-",
+ *         "version_added": "3.5",
+ *         "version_removed": "9"
+ *       }
+ *     ]
+ *   }
+ * } => "FF6"
+ */
+function supportToShortCompatString(support, browserAbbrev) {
```
Comment (connorshea) [inline build/mdn-browser-compat-data-importer.js]: This doesn't completely handle cases with more complex data, for example:

```json
{
  "support": {
    "chrome": [
      {
        "version_added": "67",
        "flags": [
          {
            "type": "preference",
            "name": "Enable Experimental Web Platform Features",
            "value_to_set": "true"
          }
        ]
      }
    ]
  }
}
```

Or:

```json
{
  "support": {
    "firefox": [
      {
        "version_added": "51",
        "version_removed": "54"
      }
    ]
  }
}
```

Maybe acceptable for an MVP, not sure :)

File: build/generate_browserjs.js (hunk 1 above)
Comment (octref) [reply build/generate_browserjs.js]: Yeah it should be BCD. Thanks.

File: build/mdn-browser-compat-data-importer.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+const bcd = require('mdn-browser-compat-data')
+
+function addBrowserCompatDataToProperties(atdirectives, pseudoclasses, pseudoelements, properties) {
+  atdirectives.forEach(item => {
+    if (bcd.css['at-rules'][item.name.slice(1)]) {
+      const matchingBCPItem = bcd.css['at-rules'][item.name.slice(1)]
```
Comment (connorshea) [inline build/mdn-browser-compat-data-importer.js]: there's also a bunch of `BCP` in this file :)

Comment (octref) [reply build/mdn-browser-compat-data-importer.js]: @connorshea You are right. Originally wanted to call it browser-compat-property.

File: build/mdn-browser-compat-data-importer.js (hunk 2 above)
Comment (octref) [reply build/mdn-browser-compat-data-importer.js]: @connorshea I opened #107 for that.

**Comment by octref** [comment]
@aeschli I updated the PR to use `all` for properties supported in all browsers, so they don't clutter the output.

One thing I left out is dropping Opera. browser-compat-data also has data for mobile browser supports such as Safari iOS, FF Android, etc. However I don't want to include all browser compat data to clutter the completion item's doc.

**Pull request merged**